lib/src/models/result_movie.dart

```diff
@@ -16,12 +18,12 @@ class ResultMovie {
 
   ResultMovie({
     this.voteCount,
-    required this.id,
+    this.id,
     this.video,
     this.voteAverage,
     this.title,
     this.popularity,
-    required this.posterPath,
```

Comment:
Why did you remove required from id and posterPath? Why the other parameters are not required?

Reply:
Because I had a problem when I wanted to show a default image when posterPath response was null, but in the end it had nothing to do with it. So I already changed all the parameters  -those who can-  to be required.